server/Makefile

```diff
@@ -130,6 +137,11 @@ LDFLAGS += -X "github.com/mattermost/mattermost/server/public/model.BuildHash=$(
 LDFLAGS += -X "github.com/mattermost/mattermost/server/public/model.BuildHashEnterprise=$(BUILD_HASH_ENTERPRISE)"
 LDFLAGS += -X "github.com/mattermost/mattermost/server/public/model.BuildEnterpriseReady=$(BUILD_ENTERPRISE_READY)"
 
+BUILD_TAGS_PARAM =
+ifneq ($(BUILD_TAGS),)
+	BUILD_TAGS_PARAM = -tags "$(BUILD_TAGS)"
```

Comment:
I tested with `-tags ""` and it works fine. So I don't think we need this. Rather simply in the Go command, we can write `-tags "$(BUILD_TAGS)"`

Reply:
Ooh, that's much more elegant -- thank you!